security-center/snippets/package.json

```diff
@@ -9,7 +9,7 @@
     "node": ">=16.0.0"
   },
   "scripts": {
-    "test": "c8 mocha -p -j 2 --recursive --timeout 6000000 system-test/v2/findings.test.js"
+    "test": "c8 mocha -p -j 2 --recursive --timeout 6000000 system-test/v2/muterule.test.js"
```

Comment:
question: Should this be testing both findings.test.js and muterule.test.js?

Reply:
Addressed.